### PR TITLE
Update README, add CODEOWNERS and PR template

### PR DESCRIPTION
**Pull request by seangoedecke**
💐

/cc @zendesk/dingo

### Description

Adds some housekeeping around gem contributions - a CODEOWNERS file, a PR template and some more up-to-date information in the README

### References
* JIRA: https://zendesk.atlassian.net/browse/

#### Before merging this PR
- [X] Fill out the Risks section
- [x] Get two approvals from the Apps teams

### Risks
* [RUNTIME] None
* None: doc only

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
+
+### Risks
+* [RUNTIME] Does this change touch the apps runtime?
+* [HIGH | medium | low] Does it work across browsers?
```
Comment (fvzzy) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Can probably remove this line

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
```
Comment (fvzzy) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: `Apps` -> `Extensibility`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
```
Comment (itirkarp) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: This is an open source repo where anyone can submit a PR so I don't think this will make sense for everyone. Maybe you could replace it with some template text like [Link to issue here if relevant]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (itirkarp) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I don't think Apps teams makes sense for an open source repo either. Besides, the rules will enforce two approvals one of which must be from a codeowner so perhaps that is enough?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
+
+### Risks
+* [RUNTIME] Does this change touch the apps runtime?
+* [HIGH | medium | low] Does it work across browsers?
+* [HIGH | medium | low] Are there any performance implications?
+* [HIGH | medium | low] Any security risks?
+* [HIGH | medium | low] What features does this touch?
```
Comment (lcampbell-zendesk) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: I wonder if these would work better as part of the checklist rather than example risks? It's pretty easy to get in the habit of just automatically deleting the bottom part of the template.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
+
+### Risks
+* [RUNTIME] Does this change touch the apps runtime?
```
Comment (itirkarp) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Will the public know what the apps runtime is? Can we reword this to `Can this change affect apps rendering for a user?`

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (seangoedecke) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: This is still an apps-specific gem, right? I wanted to try and keep this specific - API Experience are an Extensibility team, but they aren't the owners of this gem.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (seangoedecke) [reply .github/PULL_REQUEST_TEMPLATE.md:19]: Yeah, good idea.
